.github/workflows/build-and-deploy-storybook.yml

```diff
@@ -52,3 +52,17 @@ jobs:
       - run: pnpm deps
       - run: pnpm build
       - run: pnpm test
+
+  Build_Microfrontend:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - uses: pnpm/action-setup@v4
+      - run: pnpm deps
```

Comment:
We should kill off `deps` at some point since it's entirely pointless. `pnpm install` or `pnpm i` from the root installs all packages/apps anyway, no need to recursively do it